yt/visualization/eps_writer.py

```diff
@@ -929,6 +932,8 @@ def title_box(self, text, color=pyx.color.cmyk.black,
         >>> d.title_box("Halo 1", loc=(0.05,0.95))
         >>> d.save_fig()
         """
+        if text_opts is None:
+            text_opts = []
```

Comment:
```suggestion
```

Reply:
can't add None to a list, I don't think this works